ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -8,23 +8,38 @@
 
    .. cli:option:: --check
 
-      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+      Check if the input is formatted correctly.
 
-   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
+      Exits with status code 0 if input is formatted correctly.
+      Exits with status code 1 and prints a diff if formatting is not correct.
 
```

Comment:
Let's still include an example of how to call it? We do that for all rustc options, even if they are straight forward.

```suggestion
      .. code-block::

         $ rustfmt --check src/main.rs
```

Reply:
me think we should not do what was done before just because it was done before, unless of course such is required for qualification

Reply:
> we should not do what was done before just because it was done before

Good point.